lib/prometheus_exporter/server/delayed_job_collector.rb

```diff
@@ -40,6 +46,18 @@ def ensure_delayed_job_metrics
         @delayed_failed_jobs_total =
         PrometheusExporter::Metric::Counter.new(
           "delayed_failed_jobs_total", "Total number failed delayed jobs executed.")
+
+        @delayed_job_max_attempts_reached =
+            PrometheusExporter::Metric::Counter.new(
+                "delayed_job_max_attempts_reached", "Total number of delayed jobs that reached max attempts.")
```

Comment:
minor but we should keep our names consistent, see: 

https://prometheus.io/docs/practices/naming/#metric-names

Can you correct this, aside from this this is good to merge!

Reply:
Will do!